docs/kubernetes/windows-details.md

```diff
@@ -0,0 +1,335 @@
+# More on Windows and Kubernetes
+
+If you're trying to deploy Kubernetes with Windows the first time, be sure to check out the [quick start](windows.md) first. If you're looking for more details on deployments, examples or troubleshooting read on.
+
+## Customizing Windows deployments
+
+ACS-Engine allows a lot more customizations available in the [docs](../), but here are a few important ones you should know for Windows deployments. Each of these are extra parameters you can add into the ACS-Engine apimodel file (such as kubernetes-windows.json from the quick start) before running `acs-engine generate`
+
+### Changing the OS disk size
+
+The Windows Server deployments default to 30GB for the OS drive (C:), which is not enough to pull multiple `microsoft/windowsservercore` based containers. It's easiest to start with 128GB, then see what your usage is over time before shrinking it down. You can change this size by adding `osDiskSizeGB` under the `agentPoolProfiles`, such as:

```

Comment:
This phrase in the first sentence should be punctuated with a dash:
`microsoft/windowsservercore`-based containers.

Reply:
Thanks! fixed in new changelist